### PR TITLE
Notifications system

### LINKED ISSUE #25: Webhooks & notification system
- [x] custom scripts
- [x] webhooks
- [x] 3rd party services (discord/slack/pushbullet/pushover/etc)
- [ ] filter/disable notifications per drive.

### LINKED ISSUE #55: [FEAT] Test notification
I just setup scrutiny and like it. I tried to setup a notification and hopefully have included the config into docker correctly. But since my HDDs all work well at the moment (and I don't want that to change), **is there a way to provoke a notification**? Maybe on the CLI? Just to test that it'll work when it really detects something is wrong. Thanks!

### PR DESCRIPTION
**Pull request by AnalogJ**
- Note: does not support disabling notifications. Tracked by another issue.

closes #55 
closes #25

### CODE REVIEW

**Comment by kbilkis** [comment]
hey! thanks for implementing this!

I do realise this is a work in progress, but fyi - to get discord notifications working I had to comment out all urls except for `discord` and `script` for whatever reason.

**Comment by AnalogJ** [comment]
Hey @kbilkis That's expected actually. All the url's are just examples. As a user you'll need to uncomment out the services that you want to use for notifications. 
Is there a reason you expected it to work differently?

**Comment by kbilkis** [comment]
In my use case I wanted to have only the `discord` notifications, so I commented out all others, but `discord` wasn't working. After some trial-and-error I've discovered that I specifically need to uncomment the `script` 'url', which doesn't make sense at all, as it didn't even have a valid directory set I believe. But without that line `discord` doesn't work :/

**Comment by AnalogJ** [comment]
Oh, that definitely sounds like an error. Can you open a new issue for that @kbilkis ?
I want to track that bug separately from this closed PR.

**Comment by kbilkis** [comment]
> Oh, that definitely sounds like an error. Can you open a new issue for that @kbilkis ?
> I want to track that bug separately from this closed PR.

sure [issue 116](https://github.com/AnalogJ/scrutiny/issues/116)

File: README.md [hunk 1]
```diff
@@ -132,6 +132,76 @@ We support a global YAML configuration file that must be located at /scrutiny/co
...
+- Hangouts
+- IFTTT
+- Join
+- Mattermost
+- Pushbullet
+- Pushover
+- Slack
+- Teams
+- Telegram
+- Tulip
```
Comment (tianon) [inline README.md:152]: Looks like this was a typo and you probably meant Zulip here? 😅


```suggestion
- Zulip
```

**Pull request merged**